tools/sync_test_server/Dockerfile

```diff
@@ -30,4 +30,5 @@ COPY mongodb-realm-command-server.js /tmp/
 RUN npm install winston@2.4.0 temp httpdispatcher@1.0.0 fs-extra moment is-port-available@0.1.5 mongodb@4.5 mongodb-query-parser@2.4.6
 
 # Run integration test server
```

Comment:
Should we delete this line?

Reply:
Merged in a single one, still relevant as the next line boots the server.